src/main/java/aeneas/models/PuzzleLevel.java

```diff
@@ -11,10 +11,12 @@
   
   public PuzzleLevel(Bullpen bullpen) {
     super(bullpen);
+    board = new PuzzleBoard();
```

Comment:
Why would we ever need to pass in a board to a level? All levels currently create their own boards, this can be changed easily to have the board be passed in, but I'm not sure what benefit that gives